packages/@o3r/core/project.json

```diff
@@ -66,6 +66,13 @@
         "script": "prepare:publish"
       }
     },
+    "test-int": {
+      "executor": "@nrwl/jest:jest",
+      "options": {
+        "jestConfig": "packages/@o3r/core/testing/jest.config.it.js",
+        "silent": false
```

Comment:
For CI doesn't we want silent to be true ?

Reply:
updated